app/api/decorators.py

```diff
@@ -39,7 +39,10 @@ def view_function():
     """
     @wraps(func)
     def decorated_view(*args, **kwargs):
-        template, context = func(*args, **kwargs)
+        response = func(*args, **kwargs)
```

Comment:
This change seems to be unrelated to this PR. What is the motivation? `@view_function` is a deliberate abstraction requiring view functions to always return a tuple of `(template name, context object)` and this logic overrides that purpose.